src/components/Cards/Cards.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```

Comment:
The main component file **if in own component folder** should just be `index.js` i.e. `components/Cards/index.js`. This means one can just include it as `src/components/Cards` rather than `src/components/Cards/Cards`

Reply:
Let me make the change